frontends/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "ipa-frontend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next -p 7759c",
```

Comment:
```suggestion
    "dev": "next -p 7759",
```

Reply:
I'm not sure why, but it works with the `c` too. does it have any meaning I don't know about?